### ISSUE TITLE
How to add the head-extra, text-extra in polys 

### ISSUE THREAD

**Issue by charlesneimog** [cac-t-u-s/om-sharp-users#57]
Hi,

Is it possible to add extras in poly?

If I just add voices (up) in polys (down), all the extras disappear.

![image](https://user-images.githubusercontent.com/31707161/157270509-566b2a46-8ce7-4d8e-9de4-685cf8c3dc2e.png)

Thank you again!

**Comment by j-bresson** [comment]
That really loos like a bug! Can you attach your patch ?
Thanks

**Comment by charlesneimog** [comment]
Yes,

Original = [new-musicxml.zip](https://github.com/cac-t-u-s/om-sharp-users/files/8209004/new-musicxml.zip)



Adaption because om-py will not work on Mac: [new-musicxml-bug.zip](https://github.com/cac-t-u-s/om-sharp-users/files/8208997/new-musicxml-bug.zip)

Thank you Jean!

**Comment by j-bresson** [comment]
Sorry this doesn't really help: just "adding voices to poly" seem top preserve the extras:
<img width="501" alt="Screenshot 2022-03-08 at 20 29 45" src="https://user-images.githubusercontent.com/7666142/157311904-c072a456-eb0e-49b9-b4bb-9ee115a87a25.png">
or is there something else I can not see here ?

**Comment by j-bresson** [comment]
.. I think I git something though: it seems like copying the chords from a `voice` to another voice does loos extras. 🤔

**Comment by charlesneimog** [comment]
Weird, because here it does not work.

> Sorry this doesn't really help: just "adding voices to poly" seem top preserve the extras: <img alt="Screenshot 2022-03-08 at 20 29 45" width="501" src="https://user-images.githubusercontent.com/7666142/157311904-c072a456-eb0e-49b9-b4bb-9ee115a87a25.png"> or is there something else I can not see here ?

![image](https://user-images.githubusercontent.com/31707161/157315753-0fe3e504-b1f6-44f5-adbd-0f1f6489621e.png)

**Comment by charlesneimog** [comment]
On ubuntu this do not work too. 

![image](https://user-images.githubusercontent.com/31707161/157316186-13468011-98ee-4a31-92cc-fda83b0baa1d.png)

**Comment by j-bresson** [comment]
Yes, this is what I just realized: your patch has `chord-seq` objects at the top, not `voice`.
With voice it doesn't work indeed.

**Comment by charlesneimog** [comment]
Ah ok, I am sorry!

**Comment by charlesneimog** [comment]
I am using to work with `orchidea-instruments` object, in the library OM-CKN.

With this high channel I can specify some pathnames and then make some music using extended techniques of the SOL-instruments using the score. 

The next step is to specify the channels using the voice extras (that's why I saw this bug), and just import musicxml and render.

**Example piece** = https://www.youtube.com/watch?v=zq8eIPoCfFk 
**Example II** = https://youtu.be/Plmj1MEyS1s?t=936

Because of that (too) I asked you to add put all the voice values if you implement the OSC player :)...

But I have no problem with this hack, just when I play the voice (of course!).

**Comment by jnb-ableton** [comment]
> But I have no problem with this hack, just when I play the voice (of course!).

Yes, that would also crash the display of durations, for instance (because the color is picked from a set of channel colors, between 1 and 16:)

**Comment by charlesneimog** [comment]
Exactly!

**Comment by j-bresson** [comment]
> Exactly!

Sorry I removed my comment :) I said this woudl also break the display of durations (because they pick a color from 1 to 16).

**Comment by j-bresson** [comment]
Hey Charles, I think th eextra thing is fixed in om# 1.6. 
Let me know if everything works as you expect!

**Comment by charlesneimog** [comment]
Hi Jean, 

Yes, it is working very well!

![image](https://user-images.githubusercontent.com/31707161/160486931-2f6e3115-ac74-4845-b95f-315ee2c6e828.png)

Thank you for all the work in OM-Sharp!
